pyproject.toml

```diff
@@ -27,30 +27,19 @@ requires-python = ">=3.12"
 dependencies = [
     "django>=5.2",
     "django-axes>=6.5.1",
-    "django-cors-headers>=4.4.0",
-    "django-markup>=1.8.1",
     "django-jsonform>=2.22.0",
     "django-log-outgoing-requests>=0.6.1",
     "django-admin-index>=3.1.0",
-    "django-redis>=5.4.0",
```

Comment:
If we remove `django-redis`, the cache configuration in `base.py` must change to use the built in classes as well https://docs.djangoproject.com/en/5.2/topics/cache/#redis